features/ultils/world.ts

```diff
@@ -0,0 +1,24 @@
+import { World } from "@cucumber/cucumber";
+import * as cucumber from "@cucumber/cucumber";
+
+import type { SpawnSyncReturns } from "child_process";
+
+export class OurWorld extends World {
+  private _response?: SpawnSyncReturns<string>;
+
+  public get response() {
+    if (this._response === undefined) {
+      throw new Error("No response found. Please run cli-kintone first.");
+    }
+    return this._response;
+  }
+
+  public set response(value) {
+    this._response = value;
+  }
```

Comment:
I'm using a getter and setter to reduce the null check when accessing the `response`.